newton/tests/test_cable.py

```
@@ -5659,6 +5659,250 @@ def _split_cable_routes_explicit_shear_to_second_slot(test, device):
+    segment_length = 0.2
+    num_links = 6
+    k_before, k_after = 5.0e1, 5.0e2
+    frame_dt = 1.0 / 60.0
+    sim_substeps = 10
+    sim_dt = frame_dt / sim_substeps
+    num_steps = 20
+
+    def build_and_run(bend_stiffness, retarget_to=None):
+        """Build a chain at ``bend_stiffness``; if ``retarget_to`` is given, capture the
+        step graph first, THEN live-reinit joint_target_ke to it before any replay."""
+        model, state0, state1, control, rod_bodies = _build_cable_chain(
```

Ruff (0.16.2)

[warning] Unpacked variable `rod_bodies` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)